src/core/control/tools/PdfElemSelection.cpp

```diff
@@ -107,78 +70,61 @@ XojPdfPageSelectionStyle PdfElemSelection::selectionStyleForToolType(ToolType ty
     }
 }
 
+Range PdfElemSelection::getRegionBbox() const {
+    if (this->selectedTextRegion && cairo_region_num_rectangles(this->selectedTextRegion.get()) > 0) {
+        cairo_rectangle_int_t bbox;
```

Comment:
nit: to make absolutely sure it's zero initialized:
```suggestion
        cairo_rectangle_int_t bbox{};
```